x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -1391,4 +1391,202 @@ describe('when on the endpoint list page', () => {
       expect(startButton).toBeNull();
     });
   });
+  describe('endpoint list take action with RBAC controls', () => {
```

Comment:
Not needed now but I would suggest splitting this test file as it is being too big.